LightAPRS-hab/LightAPRS-hab.ino

```diff
@@ -175,7 +199,9 @@ void loop() {
 #if defined(DEVMODE)
       Serial.println(F("Not enough sattelites"));
 #endif
+        YlwLedOFF;
       }
+      YlwLedOFF;
```

Comment:
I think this is the culprit, when sleepSeconds(BeaconWait) ends it hits this, and doesn't get turned back on until after the gps ops which take some time